index.js

```diff
@@ -136,8 +136,9 @@ function renderTag(elem, opts) {
   }
 
   if (
-    opts.xmlMode
-    && (!elem.children || elem.children.length === 0)
+    (opts.xmlMode
+    && (!elem.children || elem.children.length === 0)) ||
```

Comment:
This change has no connection to `xmlMode` that is why I have a `||` condition. Tags which list under `singleTag` will have a `/>`